src/Paginator/FantaPaginatorAdapter.php

```diff
@@ -59,12 +59,9 @@ public function getSuggests()
     /**
      * Returns a slice of the results.
      *
-     * @param int $offset The offset
-     * @param int $length The length
-     *
-     * @return array|\Traversable The slice
+     * @return iterable The slice
      */
-    public function getSlice($offset, $length)
+    public function getSlice(int $offset, int $length): iterable
```

Comment:
```suggestion
    public function getSlice($offset, $length): iterable
```

And that should work as type widening is available since PHP 7.2